### PR TITLE
Release v2021.12.17-0

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: src/release-notes/v2021.12.17-0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+version: "v2021.12.14-0"
+weight: 202112140
+---
+
+### <span class="label label-green">New Features</span>
+- Added experimental support for [IPv6](https://kurl.sh/docs/install-with-kurl/ipv6) enabled hosts.
+- Added support for Containerd 1.4.8 through 1.4.12.
+
+### <span class="label label-orange">Bug Fixes</span>
+- Fix namespaces/projectcontour missing annotation warning when Contour add-on is enabled.
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:12]:
```suggestion
- Fixed missing annotation warnings for namespaces/projectcontour when the Contour add-on is enabled.
```

File: src/release-notes/v2021.12.17-0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Fix installer RPM error when migrating from Docker to Containerd.
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:13]:
```suggestion
- Fixed an installer RPM error when migrating from Docker to Containerd.
```

File: src/release-notes/v2021.12.17-0.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Fix Contour 1.7.0 error "no matches for IdId ~G_v1_Service|~X|envoy; failed to find unique target for patch ~G_v1_Service|envoy" for newer versions of Kubernetes.
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:14]:
```suggestion
- Fixed Contour 1.7.0 error, "no matches for IdId ~G_v1_Service|~X|envoy; failed to find unique target for patch ~G_v1_Service|envoy", for newer versions of Kubernetes.
```

File: src/release-notes/v2021.12.17-0.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- Fixed an issue that would cause the kURL API to fail to generate unique installer ids when changing the `kurl.hostPreflights` property.
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:15]:
```suggestion
- Fixed an issue that caused the kURL API to fail generating unique installer ids when changing the `kurl.hostPreflights` property.
```

File: src/release-notes/v2021.12.17-0.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+-
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:16]:
```suggestion
```

File: src/release-notes/v2021.12.17-0.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+### <span class="label label-green">New Features</span>
+- Added experimental support for [IPv6](https://kurl.sh/docs/install-with-kurl/ipv6) enabled hosts.
+- Added support for Containerd 1.4.8 through 1.4.12.
+
+### <span class="label label-orange">Bug Fixes</span>
+- Fixed missing annotation warnings for namespaces/projectcontour when the Contour add-on is enabled.
+- Fixed an installer RPM error when migrating from Docker to Containerd.
+- Fixed Contour 1.7.0 error, "no matches for IdId ~G_v1_Service|~X|envoy; failed to find unique target for patch ~G_v1_Service|envoy", for newer versions of Kubernetes.
+- Fixed an issue that caused the kURL API to fail generating unique installer ids when changing the `kurl.hostPreflights` property.
```
Comment (samantha-replicated) [inline src/release-notes/v2021.12.17-0.md:15]:
```suggestion
- Fixed an issue that caused the kURL API to fail generating unique installer IDs when changing the `kurl.hostPreflights` property.
```

**Review by samantha-replicated — APPROVED** [review]
Content is good, just have to troubleshoot the weird deploy error with Netlify. Seems to be an issue with Gatsby.

**Comment by emosbaugh** [comment]
This should fix the deploy error https://github.com/replicatedhq/kurl.sh/pull/631/commits/432b29f9ab120711db3bec8ef19dbf39dd0429ac

**Pull request merged**